src/components/LandingPageBlocks/HeroBlock/HeroBlock.js

```diff
@@ -1,14 +1,73 @@
-import React from 'react';
+import React,{useEffect} from 'react';
+import background from "../../../image/bg.png";
+import working from "../../../image/working.png";
+import arrow1 from "../../../image/Arrow1.png";
+import arrow2 from "../../../image/Arrow2.png";
+import mobileArrowLeft from "../../../image/mobileArrowLeft.png";
+import mobileArrowRight from "../../../image/mobileArrowRight.png";
+import mobileBg from "../../../image/mobileBg.png";
 import './HeroBlock.css';
+import {startAnimation} from './animationScripts/animationSript';
 
 const HeroBlock = props => {
+
+	useEffect(() => {
+        startAnimation()
+    })
+
     return (
-        <section className="HB-Section">
-            <content className="HB-content">
-                <h1 className="HB-title">HERO BLOCK</h1>
-            </content>
-        </section>
+        <section>
+
+			 <img src={background} className="large_image" alt=""/> 
```

Comment:
What's wrong with indentation here? In some places there are spaces, in some are tabs. This should be unified. I'm pretty sure that should be plugins for JSX formatting for VSC.